src/components/dialogs/parameters/sensi/sensitivity-table.tsx

```diff
@@ -68,45 +64,36 @@ const SensitivityTable: FunctionComponent<SensitivityTableProps> = ({
                 border: 'solid 0px rgba(0,0,0,0.1)',
             }}
         >
-            <Table stickyHeader size="small">
+            <Table stickyHeader size="small" sx={{ tableLayout: 'fixed' }}>
                 <TableHead>
-                    <TableRow>
-                        {columnsDefinition.map((column: any) => (
-                            <TableCell>
-                                <Box
-                                    sx={{
-                                        backgroundColor: column.color,
-                                    }}
-                                >
-                                    <FormattedMessage id={column.label} />
-                                </Box>
-                            </TableCell>
-                        ))}
-                        <TableCell>
-                            <Tooltip
-                                title={intl.formatMessage({
-                                    id: 'AddRows',
-                                })}
+                    {columnsDefinition.map((column: any) => (
```

Comment:
column have the type  IColumnsDef (you can added also to SensitivityTableProps)

Reply:
done